AnyText/AnyText.Core/Rules/MultiRuleApplication.cs

```diff
@@ -161,6 +163,28 @@ public override object GetValue(ParseContext context)
             }
         }
 
+        /// <inheritdoc />
+        public override void AddDocumentSymbols(ParseContext context, ICollection<DocumentSymbol> result)
+        {
+            if (Rule is OneOrMoreRule)
```

Comment:
Bitte hier eine virtuelle Methode in Rule, ob die DocumentSymbols weitergereicht werden soll, das wird hier zu kompliziert sonst. Es gibt auch keinen sinnvollen Grund, warum das bei einer ZeroOrMoreRule oder ZeroOrOneRule anders sein sollte.